cclient/api/core/attach_to_tangle.h

```diff
@@ -26,28 +26,30 @@ extern "C" {
 #endif
 
 /**
- * @brief
+ * @brief Performs Proof-of-Work required to attach a transation to the Tangle.
+ *
+ * <p>
+ * Prepares the specified transactions (trytes) for attachment to the Tangle by doing Proof of Work. You need to supply
```

Comment:
Suggestion - 
"You will have to" in "You need to"

Reply:
Suggestion -
"These are the tips which you're going to validate and
 * reference with this transaction." --> "The new transaction is referencing and validating these new tips"

Reply:
There are no new transaction and new tips, it's this transaction and tips selected from the IRI node.